src/Orc.Controls/Controls/NumericTextBox/NumericTextBox.cs

```diff
@@ -343,13 +343,6 @@ private static bool DoesStringValueRequireUpdate(string text)
                 update = false;
             }
 
-            if (text.StartsWith(CommaCharacter) || text.EndsWith(CommaCharacter) ||
```

Comment:
Are you sure this can be removed? Are we not breaking existing logic?

Reply:
The same code blocks repeated twice